src/components/pages/DashboardPage/Dashboard/Dashboard.tsx

```diff
@@ -6,26 +6,29 @@ import { UserData } from '../../../../types';
 import { setupMaster } from 'cluster';
 import { userInfo } from 'os';
 
-const Dashboard = (dbprops: { id: number, name: string, setUser: any, user: UserData }) => {
+interface IUser {
+  id: number,
+  currUser: UserData
+}
 
-  console.log('setUser method?', dbprops.user.first_name);
+const Dashboard = ({ id, currUser }: IUser) => {
+  const [allFilms, setAllFilms] = useState<object[]>([]);
 
   useEffect(() => {
-
-
-    // dbprops.setUser({
-    //   "email": "rachelmaria@gmail.com",
-    //   "first_name": "Nina",
-    //   "last_name": "Brissey"
-    // })
+    fetch(`https://epk-be.herokuapp.com/api/v1/users/${id}`)
+      .then(res => res.json())
+      .then((data: any) => {
```

Comment:
Anything that we type as any, let's do a command F when we have time and run over these with a mentor.